exec/java-exec/src/main/java/org/apache/drill/exec/server/rest/StatusResources.java

```diff
@@ -183,9 +188,9 @@ public Viewable getSystemInternalOptions(@Context UriInfo uriInfo) {
   public Viewable updateSystemOption(@FormParam("name") String name,
                                    @FormParam("value") String value,
                                    @FormParam("kind") String kind) {
-    SystemOptionManager optionManager = work.getContext()
-      .getOptionManager();
-
+    SystemOptionManager optionManager = authEnabled.separateWorkspace()
```

Comment:
This seems to say that updating a *system* option works differently depending on whether the user has their own workspace. Shouldn't *system* options always update the (global) system options? Otherwise, do we need a "real system option" attribute to really update the global options?